package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "reactjs-mentoring",
+  "version": "1.0.0",
+  "description": "ReactJS Global Mentoring Program 2020Q3",
+  "private": true,
+  "main": "index.jsx",
+  "dependencies": {
+    "cross-env": "^7.0.2",
```

Comment:
not critical but looks like it is dev dependency

Reply:
Initially, I thought as you. But we use env variable for prod build. Maybe I'm wrong.

Reply:
discussed in teams. fixed.